src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -10639,6 +10639,96 @@ namespace WaterThermalTanks {
         return PLRResidualHPWH;
     }
 
+    bool SourceHeatNeed(int const WaterThermalTankNum,
+                        Real64 const OutletTemp,
+                        Real64 const DeadBandTemp,
+                        Real64 const SetPointTemp
+        ){
+        // FUNCTION INFORMATION:
+        //       AUTHOR         Yueyue Zhou
+        //       DATE WRITTEN   May 2019
+        //       MODIFIED       na
+        //       RE-ENGINEERED  na
+
+        // PURPOSE OF THIS FUNCTION:
+        // Determine by tank type, tank temperature and control mode if source side flow is needed
+
+        // METHODOLOGY EMPLOYED:
+        // <description>
+
+        // REFERENCES:
+        // na
+
+        // Using/Aliasing
+        using ScheduleManager::GetCurrentScheduleValue;
```

Comment:
I'd prefer you actually include `ScheduleManager::GetCurrentScheduleValue` in the code below rather than having a `using` statement at the top of the function.  In this case, it's not a big deal, but in many places within E+, the usage of `using` is huge and causes uncertainty around where a function or variable is being accessed.